src/Confluent.Kafka/Consumer.cs

```diff
@@ -62,6 +62,7 @@ internal class Config
 
         private int cancellationDelayMaxMs;
 
+        private bool subscribeHasBeenCalled = false;
```

Comment:
needs locking

Reply:
i omitted this on purpose and was half expecting a comment :-) - for this to ever come into play, the user would need to have code that is so screwed up that any problem they had would be un-solvable by adding a lock here. but ok, sure.

Reply:
btw: i'm proactively trying to be less rude in PR comments when there's absolutely no reason to be. failing, apparently.